File: SeekMatch-Client/src/app/shared/models/representative.ts
import { Setting } from "./setting";
import { User } from "./user";

export class Representative extends User {

  public firstName: string = '';
  public lastName: string = '';
  public password: string = '';
  public position: string = '';
  public setting: Setting = new Setting();

  constructor(init?: Partial<Representative>) {
    super();
    Object.assign(this, init);
  }
}

File: SeekMatch-Server/SeekMatch.Core/Entities/Representative.cs
﻿namespace SeekMatch.Core.Entities
{
    public class Representative : BaseEntity
    {
        public required string FirstName { get; set; }
        public required string LastName { get; set; }
        public string? Position { get; set; }
        public required User User { get; set; }
        public required Company Company { get; set; }
        public string? CompanyId { get; set; }

    }
}
